.github/workflows/cypress.yml

```diff
@@ -107,6 +107,74 @@ jobs:
           CI: "true"
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+  cypress-react:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    strategy:
+      # when one test fails, DO NOT cancel the other
+      # containers, because this will kill Cypress processes
+      # leaving the Dashboard hanging ...
+      # https://github.com/cypress-io/github-action/issues/48
+      fail-fast: false
+      matrix:
```

Comment:
I don't think we need a matrix at all for just running 1 copy of the job, this whole `strategy` bit can be removed

Reply:
nope, because after refactor we will still use those 20 machines

Reply:
Yeah fair enough actually